src/components/Friendly-captcha/Frontend/useCaptchaHook.tsx

```diff
@@ -19,6 +26,9 @@ type FriendlyCaptchaWidgetProps = Required<FriendlyCaptchaProps> &
   React.HTMLAttributes<HTMLDivElement> & {
     solvedHandler: (solution: string) => void;
     errorHandler: (solution: string) => void;
+    resetHandler: () => void;
+    captchaRendered: boolean;
+    setCaptchaRendered: Dispatch<SetStateAction<boolean>>;
   };
 
 type CaptchaStatus = {
```

Comment:
Maybe set up a type for error? I believe it is a string, but please double-check